File: ruby/json_data.rb
require 'oauth'
require 'json'

c = OAuth::Consumer.new('f073e5b4-e638-439a-8109-7da713cfd73e', '2bc1JjFYqGEqRVaK1RsHnbeiVtgnASHl', {:site => 'https://serwisant-online.pl', :signature_method => "HMAC-SHA1"})

result = c.request(:get, '/api/v1/orders/1425.json')

raise "HTTP request error - #{result.code}" if result.code.to_i != 200

order = JSON.parse(c.request(:get, '/api/v1/orders/1425.json').body)

raise "App request error #{order['errors']}" if order.key?('errors')

puts order.inspect

File: php/json_data.php
<?php
include 'vendor/autoload.php';

use Serwisant\SerwisantApi;

$consumer = new SerwisantApi\ConsumerOauth('f073e5b4-e638-439a-8109-7da713cfd73e', '2bc1JjFYqGEqRVaK1RsHnbeiVtgnASHl');

$order = $consumer->get('/api/v1/orders/1425');
$orders = $consumer->get('/api/v1/orders', ['filter' => 'open']);

var_dump($order);
var_dump($orders);
